src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,34 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int dozerWidth;
+    private int dozerHeight;
```

Comment:
Don't forget about access modifiers for your custom fields. It's a good practice to make fields private and provide public getters and setters for access.